src/esp/gfx/CMakeLists.txt

```diff
@@ -26,6 +28,8 @@ set(
   RenderTarget.h
   ShaderManager.cpp
   ShaderManager.h
+  PBRShader.cpp
+  PBRShader.h
```

Comment:
Just FYI, Magnum's coding style uses `Pbr` instead of `PBR` (e.g. [Trade::PbrMetallicRoughnessMaterialData](https://doc.magnum.graphics/magnum/classMagnum_1_1Trade_1_1PbrMetallicRoughnessMaterialData.html)), all acronyms are capitalized this way. I don't _demand_ you to change this, just nothing that so you are aware of the difference.